.github/workflows/release-cli.yml

```
@@ -18,7 +18,9 @@ on:
 
 jobs:
   build-and-package:
-    runs-on: ubuntu-22.04
+    # Use ubuntu-20.04 for maximum Linux compatibility (glibc 2.31)
+    # Binaries built here work on: Debian 10+, Ubuntu 20.04+, RHEL 8+
+    runs-on: ubuntu-20.04
```

actionlint (1.7.9)

label "ubuntu-20.04" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xl", "macos-latest-xlarge", "macos-latest-large", "macos-26-xlarge", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xl", "macos-14-xlarge", "macos-14-large", "macos-14", "macos-13-xl", "macos-13-xlarge", "macos-13-large", "macos-13", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)